python/mxnet/gluon/block.py

```diff
@@ -1225,6 +1226,7 @@ def hybridize(self, active=True, backend=None, backend_opts=None, **kwargs):
             The name of backend, as registered in `SubgraphBackendRegistry`, default None
         backend_opts : dict of user-specified options to pass to the backend for partitioning, optional
             Passed on to `PrePartition` and `PostPartition` functions of `SubgraphProperty`
+        clear : clears any previous optimizations
```

Comment:
Is there a case where we want to call `hybridize(clear=False)`?

Reply:
Yes, Previously hybridize used to clear the cachedOp and cached_graph on every call. In this PR we add a new argument clear that defaults to True to preserve current behavior. When set to False it allows re-using the previous cached_graph to chain multiple optimization calls together. Heres an example:
```
net = nn.SymbolBlock(sym, inputs)
net.optimize_for(x, backend="opt1")
net.optimize_for(x, backend="opt2", clear=False)
```

Reply:
Maybe the 1st call to `optimize_for` is to partition the graph, and the 2nd call would be to run a custom graph pass for example.

Reply:
cool got it